src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -91,7 +104,7 @@ class SideNav extends Component {
 
       return (
         <li className={`va-sidenav-level-${depth}`} key={id}>
-          <button
+          <div
```

Comment:
![image](https://user-images.githubusercontent.com/12773166/67326803-04661d80-f4e5-11e9-990f-c1e9fd697645.png)

Hmm, so I wanted to make this a `button` but looks like `react-dom` is complaining about that. Any suggestions @ncksllvn since I know you warned about having an `onClick` listener on a `div` tag?